.github/workflows/push_pr.yml

```diff
@@ -20,13 +20,18 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
```

Comment:
why do we need to setup go now?

Reply:
I think some of the linters needed it, but I have to admit I can't remember the details. I took it from [newrelic-prometheus](https://github.com/newrelic-forks/newrelic-prometheus/blob/main/.github/workflows/push_pr.yaml#L139).

Reply:
TIL: Due to this https://github.com/golangci/golangci-lint-action/pull/403